heat/cluster/tests/test_spectral.py

```diff
@@ -0,0 +1,112 @@
+import os
+import unittest
+
+import heat as ht
+
+if os.environ.get("DEVICE") == "gpu" and ht.torch.cuda.is_available():
```

Comment:
Does this now have to be in any test case class? If so we should probably factor this out into our own test class

Reply:
yes, good idea. Will open an issue. untill then....

Reply:
See #523